src/v/coproc/api.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "coproc/event_listener.h"
```

Comment:
You don't need this include, but you'll probably have to declare the destructor in the header: `api::~api();` and define it in the cpp `api::~api() = default`

Reply:
Awesome! It worked! Thanks!